spring-cloud-gcp-samples/spring-cloud-gcp-data-firestore-sample/pom.xml

```diff
@@ -65,4 +83,97 @@
       </plugin>
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>native</id>
+      <properties>
+        <integration-test.tags.include>native</integration-test.tags.include>
+        <integration-test.tags.exclude/>
+        <!-- Customize Spring Boot classifier, useful in order to avoid a clash between Spring Boot repackaging and native-image-maven-plugin -->
+        <classifier/>
+        <!-- Customize native-image arguments with this property -->
+        <native.build.args>
+          --enable-http
+          --enable-https
+        </native.build.args>
+
+        <!-- `tiny` builder allows small footprint and reduced surface attack, you can also use `base` (the default) or `full` builders to have more tooling available in the image for an improved developer experience -->
+        <!-- TODO Bring back "paketobuildpacks/builder:tiny" before 0.10.0 release -->
+        <builder>dmikusa/graalvm-tiny</builder>
```

Comment:
Sorry not too familiar with paketo; what's the reason for using `dmikusa/graalvm-tiny` here over `paketobuildpacks/builder:tiny`?

Reply:
sorry! forgot to change this :neutral_face:

Reply:
Great, thanks!